src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonDocument.Parse.cs

```diff
@@ -12,6 +12,11 @@ namespace System.Text.Json
 {
     public sealed partial class JsonDocument
     {
+        // Cached unrented documents for literal values.
+        private static JsonDocument? s_nullLiteral;
```

Comment:
Are we allowed to just assign the static right up front, or is there some value in doing it only at runtime (with a null check, below) on the first call to Parse.
```C#
private static JsonDocument s_nullLiteral = Create(JsonConstants.FalseValue.ToArray());
```

Reply:
We could, but it would increase startup perf for the standard (non serializer) code path. Also, the "null" literal isn't actually even used by the serializer, only when the new API is called manually.